publish_distro

```diff
@@ -70,6 +70,10 @@ for arg do
         do_cleanup=1
         date="90 days ago"
         continue
+  elif [ "$arg" == '--now' ]; then
+	PUBLISH_DELAY=0
```

Comment:
I get the feeling the indentation is broken - or you used tabs, which is just as bad

Reply:
Good catch! I fixed this particular line, otherwise I'll do a separate PR with retab of the rest of the file. Seems like it was mostly using tabs.